src/app/jscaip/state/GameState.ts

```diff
@@ -1,7 +1,30 @@
+import { MGPOptional } from '@everyboard/lib';
+
 import { Player } from '../Player';
+import { RulesConfig } from '../RulesConfigUtil';
+
+export type GameStateAndConfig = GameState | { state: GameState; config: RulesConfig };
```

Comment:
uuuw, GameStateAndConfig qui est GameStateOrGameStateAndConfig.
Je propose GameStateAndConfig qui est ce qu'il dit: une paire de state et de config ! Et qui a les accesseurs utiles pour extraire ce qu'on souhaite

Reply:
should be good now !